DSharpPlus/Net/RestClient.cs

```diff
@@ -123,7 +124,7 @@ private async Task ExecuteRequestAsync(BaseRestRequest request, RateLimitBucket
                     if (Interlocked.Decrement(ref bucket._remaining) < 0)
 #pragma warning restore 420 // blaze it
                     {
-                        request.Discord?.DebugLogger?.LogMessage(LogLevel.Debug, "REST", $"Request for {bucket}. Blocking.", DateTime.Now);
+                        request.Discord?.Logger?.LogDebug(BaseDiscordClient.RestEventId, "Request for {0} is blocked", bucket.ToString());
```

Comment:
Minor thing, all of these have `Logger?.` and they're the only places that assume the ILogger instance can be null. I believe those are a leftover from the old code and only the leftmost null-conditional case needs to be here.

Reply:
I recall entire chain requiring it, but maybe that changed.